docs/README.rst

```diff
@@ -74,7 +74,24 @@ Once you have built the documentation run the following command from the root di
 
     docs/start_doc_server.sh
 
-Then, view your docs at ``localhost:8000``
+You need to have python installed to run the command, if you get an error message -- python: command not found
+It is possible that you don't have python installed or added to PATH.
+Visit link <https://python.land/installing-python> to install python and add to PATH
```

Comment:
```suggestion
You need to have python installed to run the command, if you get an error message ``-- python: command not found``
It is possible that you don't have python installed or added to PATH.
Visit `link <https://python.land/installing-python>`_ to install python and add to PATH
```

Reply:
I think the `--` is not a part of the message? It should be

```rst
if you get an error message -- ``python: command not found``
```

instead.

Also please capitalise *python* except in code blocks.

Reply:
ok

Reply:
ok